.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "test/unit/kpatch-unit-objs"]
+	path = test/unit/kpatch-unit-objs
```

Comment:
Can we make the path `test/unit/objs`?  Easier to type and less redundant.

Reply:
Yeah, sounds fine to me.  Or even "kpatch-unit-test-objs" if that's not too verbose.